app/views/pages/report.ejs

```diff
@@ -132,7 +128,7 @@
         <h1><%= profile ? 'Din helsetilstand' : 'Rapporter din helsetilstand' %></h1>
         <% if(locals.success) {%>
           <div class="alert alert-success" role="alert">
-            Takk for ditt svar! Benytt denne lenken hvis helsetilstanden din endrer seg: <a target="_blank" href="/helsetilstand/<%= passcode %>">coronastatus.no/helsetilstand/<%= passcode %></a>. 
+            Takk for ditt svar! Benytt denne lenken hvis helsetilstanden din endrer seg: <a target="_blank" href="<%= urls.profile %>/<%= passcode %>">coronastatus.no/helsetilstand/<%= passcode %></a>. 
```

Comment:
missed one in the <a> content here :) or should that be fully translated separately?

edit: Lets keep all local urls outside of translation files.

Reply:
Seems like I forgot a few 😅 Think I got them all now, https://github.com/BustByte/coronastatus/pull/199/commits/0077913f275611884107f7380b43039ba6e51ca4